cmd/pull.go

```diff
@@ -147,8 +150,11 @@ func (p *PullParams) Validate(ctx ActionCtx) error {
 	if err != nil {
 		return err
 	}
-	if oc.AccountServerURL == "" {
-		return fmt.Errorf("operator %q doesn't set account server url - unable to pull", ctx.StoreCtx().Operator.Name)
+	if oc.AccountServerURL == "" && len(oc.OperatorServiceURLs) == 0 {
+		return fmt.Errorf("operator %q doesn't set account server url or operator service url - unable to pull", ctx.StoreCtx().Operator.Name)
+	}
+	if oc.AccountServerURL == "" && len(oc.OperatorServiceURLs) > 0 && !p.All {
+		return fmt.Errorf("operator %q can only pull all jwt - unable to pull by account", ctx.StoreCtx().Operator.Name)
```

Comment:
So we are mixing the intent - yes, service URLs is the same host:port where we connect, but AccountServerURL is saying this is where you connect to perform JWT updates. By honoring that (only looking at AccountServerURL) all this interdependency is dropped. I would stutter here and have them specify the intention that JWT updates happen at the specified URL (even if it duplicates an OperatorServiceURL).

Reply:
The problem is that the account server url does not enforce a schema. 
So I went with the service url, as this means you will be able to access nats here. (thats' all we need)

I follow the general pattern of making sure no account JWT Server url is set, it would imply an accounts server.
Only then do I assume push/pull want to connect nats-server

```
nsc describe operator
╭──────────────────────────────────────────────────────────────────────────────────╮
│                                 Operator Details                                 │
├───────────────────────┬──────────────────────────────────────────────────────────┤
│ Name                  │ testme                                                   │
│ Operator ID           │ OCLA4VYWAFUJE4BCYI2SI277W6SXX7FCIFXC6JZUUN333TMTYCXFCSUH │
│ Issuer ID             │ OCLA4VYWAFUJE4BCYI2SI277W6SXX7FCIFXC6JZUUN333TMTYCXFCSUH │
│ Issued                │ 2020-10-13 15:30:09 UTC                                  │
│ Expires               │                                                          │
│ Account JWT Server    │ localhost:4222                                           │
│ Operator Service URLs │ nats://localhost:4222                                    │
│ System Account        │ AACTUIFJ3D2HCUCPAW2I3D44OM4JIG2HQSHV6GFI5ZM7RSKK2YN2PCGO │
╰───────────────────────┴──────────────────────────────────────────────────────────╯
```

Reply:
I think the issue here is intent. Yes you can do it there, but maybe that is not an indication that the management is configured or desired. The AccountServerURL would provide for such intent.